src/app/fdctl/config/default.toml

```
@@ -591,6 +586,13 @@ dynamic_port_range = "8900-9000"
         # so this should be unused.
         tx_buf_size = 4096
 
+        # QUIC has a concept of an "idle" connection - one where neither the
```

This is a major nit but for this user facing documentation I really want to keep a consistent style. Can you reformat this to match the rest of the file? Double spaced sentences. Double newline between paragraphs, wrap to 72 etc. I would format this as

```suggestion
        # QUIC has a concept of an idle connection, one where neither
        # the client nor the server has sent any packet to the other for
        # a period of time.  Once this timeout is reached the connection
        # will be terminated.
```